packages/blockchain-info-components/src/Colors/DarkMode.ts

```diff
@@ -24,7 +24,7 @@ const DarkTheme: DefaultTheme = {
   grey700: lighten(0.1, '#50596B'),
   grey800: lighten(0.1, '#B1B8C7'),
   grey900: '#121D33',
-  blue000: '#ECF5FE',
+  blue000: '#1c1c1c',
```

Comment:
Intended?

Reply:
yeah, the lighter background looked odd in dark mode, so switched it to a darker color. checked other places that color was used, looked good too.

Reply:
👌